package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "babel": "^5.8.21",
-    "babel-eslint": "^4.1.3",
+    "babel-eslint": "^4.1.8",
```

Comment:
Is there a reason to update this version? (ie. is there a bug it fixes that we should know about?)